tests/nested/manual/core20-early-config/task.yaml

```diff
@@ -74,13 +74,13 @@ execute: |
         remote.exec "hostname" | MATCH "foo"
 
         # netplan config defaults are applied
-        remote.exec "cat /etc/netplan/0-snapd-defaults.yaml" | MATCH br54
-        remote.exec "netplan get bridges.br54.dhcp4" | MATCH true
+        remote.exec "sudo cat /etc/netplan/0-snapd-defaults.yaml" | MATCH br54
```

Comment:
Please see https://github.com/snapcore/snapd/pull/12323 and https://github.com/snapcore/snapd/pull/12376